app/Actions/User/GetUserInfoPresenter.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Hedonist\Actions\User;
+
+use Hedonist\Actions\Presenters\User\UserPresenter;
+
+class GetUserInfoPresenter
+{
+    private $userPresenter;
+
+    public function __construct(UserPresenter $presenter)
+    {
+        $this->userPresenter = $presenter;
+    }
+
+    public function present(GetUserInfoResponse $response)
+    {
+        $result = $this->userPresenter->present($response->getUser());
+        $result['followers'] = $this->userPresenter->presentCollection($response->getUser()->followers);
```

Comment:
2 more queries to the db getUser()->followers, pass these values to GetUserInfoResponse

Reply:
In this case it doesn't really matters since there are no n+1 problem, queries amount will be equal in any case. But ok, I really should fix it.